fMRIVolume/GenericfMRIVolumeProcessingPipeline.sh

```diff
@@ -342,57 +385,82 @@ fi
 
 log_Msg "mkdir -p ${fMRIFolder}/MotionCorrection"
 mkdir -p "$fMRIFolder"/MotionCorrection
+
+if [ ! $fMRIReference = "NONE" ] ; then
+    log_Msg "Using reference image from ${fMRIReference}"
+    reference="$fMRIReference"/"$ScoutName"_gdc
```

Comment:
Should there be a check here that `${reference}` actually exists?
Also, should there be a check that `${fMRIReference}` and `${fMRIFolder}` aren't in fact the same thing?

Reply:
Added this check as well.